authors/apps/authentication/renderers.py

```diff
@@ -12,6 +12,7 @@ def render(self, data, media_type=None, renderer_context=None):
         # the default JSONRenderer to handle rendering errors, so we need to
         # check for this case.
         errors = data.get('errors', None)
+        print(errors)
```

Comment:
Remove print

Reply:
You could check for print debug statements using `Command + F` before committing.